DxDispatch/src/dxdispatch/Device.cpp

```diff
@@ -171,7 +171,7 @@ Device::Device(
     THROW_IF_FAILED(m_dmlModule->CreateDevice1(
         m_d3d.Get(), 
         dmlCreateDeviceFlags, 
-        DML_FEATURE_LEVEL_5_0, 
+        DML_FEATURE_LEVEL_6_2, 
```

Comment:
Do we need to leave the version at 5.0?

Reply:
It should be whatever version of DML ORT is meant to be paired with, and if we use *any* DML 6.2 operators (in OperatorRegistration.cpp `operatorRegistrationInformationTable`), then it needs to be 6.2.

Reply:
Sorry, my previous reply was nonsense, because I had too many tabs open and thought this was in the ORT repo. :b

I think this should be left at whatever the minimum viable DML version is. Justin would have more of an opinion.